index.d.ts

```diff
@@ -0,0 +1,12 @@
+type ClassValue = ClassArray | ClassDictionary | string | number | null | boolean;
```

Comment:
Could we export this type as well please?

Reply:
Sure can! :D